src/components/NewsletterForm/NewsletterFormResearchDropDown.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import NewsletterFormItem from './NewsletterFormItem';
+
+type NewsletterFormResearchDropDownProps = {
+  handleChange?: (event: React.FormEvent<HTMLSelectElement>) => void;
+  value: string;
+};
+
+export const NewsletterFormResearchDropDown = ({
+  handleChange,
+  value
+}: NewsletterFormResearchDropDownProps) => {
+  return (
+    <NewsletterFormItem type="dropdown">
+      <label
+        className="newsletter-form__item-label newsletter-form__item-label--dropdown"
+        htmlFor="dropdown"
```

Comment:
`htmlFor` is react equivalent of `for` attribute in HTML and so should reference the id of the associated component

in this case this should be `edit-research-dd`